mods/sethome/init.lua

```diff
@@ -54,11 +54,13 @@ minetest.register_chatcommand("sethome", {
         minetest.chat_send_player(name, "Home set!")
         changed = true
         if changed then
-        	local output = io.open(homes_file, "w")
-            for i, v in pairs(homepos) do
-                output:write(v.x.." "..v.y.." "..v.z.." "..i.."\n")
-            end
-            io.close(output)
+        	local output = io.open(home_file, "w")
+			local data = {}
```

Comment:
I think my editor did that opps lol
On Dec 22, 2015 3:10 AM, "asl97" <notifications@github.com> wrote:

> In mods/sethome/init.lua
> <https://github.com/minetest/minetest_game/pull/752#discussion_r48230026>:
>
> > @@ -54,11 +54,13 @@ minetest.register_chatcommand("sethome", {
> >          minetest.chat_send_player(name, "Home set!")
> >          changed = true
> >          if changed then
> > -        	local output = io.open(homes_file, "w")
> > -            for i, v in pairs(homepos) do
> > -                output:write(v.x.." "..v.y.." "..v.z.." "..i.."\n")
> > -            end
> > -            io.close(output)
> > +        	local output = io.open(home_file, "w")
> > +			local data = {}
>
> why the indent?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/minetest/minetest_game/pull/752/files#r48230026>.
>